src/containers/SearchPage/SearchContainer.jsx

```diff
@@ -6,260 +6,151 @@
-  const searchResults = isReadyToShow
-    ? convertResult(search.results, searchParams.subjects, enabledTab, locale)
-    : [];
   return (
-    <SearchPage
-      closeUrl="/#"
-      searchString={query || ''}
-      onSearchFieldChange={setQuery}
-      onSearch={onQuerySubmit}
-      onSearchFieldFilterRemove={onSearchFieldFilterRemove}
-      searchFieldFilters={activeSubjectsMapped}
-      activeFilters={activeSubjectsMapped}
-      messages={searchPageMessages(resultMetadata.totalCount)}
-      resourceToLinkProps={resourceToLinkProps}
-      filters={searchFilters}>
-      <SearchResults
-        results={searchResults}
-        resourceTypes={data && data.resourceTypes ? data.resourceTypes : []}
-        loading={loading}
-        enabledTab={enabledTab}
-        resultMetadata={resultMetadata}
-        searchParams={searchParams}
-        enabledTabs={enabledTabs}
-        allTabValue={allTabValue}
-        onTabChange={updateTab}
-        query={searchParams.query}
-        onUpdateContextFilters={onUpdateContextFilters}
-        includeEmbedButton={includeEmbedButton}
-        ltiData={ltiData}
-        isLti={isLti}
+    <>
+      <SearchHeader
+        searchPhrase={query}
+        searchPhraseSuggestion={suggestion}
+        searchPhraseSuggestionOnClick={() =>
+          handleSearchParamsChange({ query: suggestion })
+        }
+        searchValue={searchValue}
+        onSearchValueChange={value => setSearchValue(value)}
+        onSubmit={handleSearchSubmit}
+        activeFilters={{
+          filters: activeSubjectFilters,
+          onFilterRemove: handleFilterRemove,
+        }}
+        filters={filterProps}
       />
-      {isReadyToShow && (
-        <Pager
-          page={searchParams.page ? parseInt(searchParams.page, 10) : 1}
-          lastPage={resultMetadata.lastPage}
-          query={searchParams}
-          pathname=""
-          onClick={updateFilter}
-          pageItemComponentClass="button"
+      {showConcepts && concepts?.length ? (
+        <SearchNotionsResult
+          items={concepts}
+          totalCount={concepts.length}
+          onRemove={() => {
+            setShowConcepts(false);
+          }}
         />
-      )}
-    </SearchPage>
+      ) : null}
+      <SearchSubjectResult items={subjectItems} />
+      <FilterTabs
+        dropdownBtnLabel="Velg"
+        value={currentSubjectType ? currentSubjectType : 'ALL'}
+        options={filterTypeOptions(searchGroups, t)}
+        contentId="search-result-content"
+        onChange={handleSetSubjectType}>
+        <SearchResults
+          searchGroups={searchGroups.sort(
+            (a, b) =>
+              sortedResourceTypes.indexOf(a.type) -
+              sortedResourceTypes.indexOf(b.type),
+          )}
+          currentSubjectType={currentSubjectType}
+          typeFilter={typeFilter}
+          handleFilterClick={handleFilterClick}
+          handleShowMore={handleShowMore}
+        />
+      </FilterTabs>
```

Comment:
FilterTabs komponenten er gammel og langt fra optimal i frontend-packages. Den bør skrives om, men det er en større jobb. Problemet med den er at den ikke takler oppdatering av options. 
Kodeforslaget mitt her sørger for at den ikke rendres før søkeresultatet er klart, dvs slik at at alle options er klare når FilterTabs rendres første gang.
```suggestion
      {searchGroups.length > 0 && (
        <FilterTabs
          dropdownBtnLabel="Velg"
          value={currentSubjectType ? currentSubjectType : 'ALL'}
          options={filterTypeOptions(searchGroups, t)}
          contentId="search-result-content"
          onChange={handleSetSubjectType}>
          <SearchResults
            searchGroups={searchGroups.sort(
              (a, b) =>
                sortedResourceTypes.indexOf(a.type) -
                sortedResourceTypes.indexOf(b.type),
            )}
            currentSubjectType={currentSubjectType}
            typeFilter={typeFilter}
            handleFilterClick={handleFilterClick}
            handleShowMore={handleShowMore}
          />
        </FilterTabs>
      )}
```

Reply:
Fortsatt litt problemer med denne. Gå f.eks til: https://ndla-frontend-pr-599.ndla.sh/search?query=abbor og søk på "historie". Ressurstypene som ikke finnes for abbor havner da i nedtrekksmenyen.

Reply:
Er dette viktig nok til at det skal gjøres noe med? Tror at det må fikses i FilterTabs komponenten i frontend-packages